graylog2-web-interface/src/components/indices/rotation/TimeBasedSizeOptimizingStrategySummary.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import { Pluralize } from 'components/common';
+
+import type { TimeBasedSizeOptimizingStrategyConfig } from './TimeBasedSizeOptimizingStrategyConfiguration';
+import { durationToRoundedDays } from './TimeBasedSizeOptimizingStrategyConfiguration';
+
+const TimeBasedSizeOptimizingStrategySummary = ({ config: { index_lifetime_hard, index_lifetime_soft } }: {config: TimeBasedSizeOptimizingStrategyConfig}) => {
+  const softLifetimeAsDays = durationToRoundedDays(index_lifetime_soft);
+  const hardLifetimeAsDays = durationToRoundedDays(index_lifetime_hard);
```

Comment:
Since we changed the name to min/max lifetime in the backend, let's do it here too.

Reply:
This seems to work for me:
```
diff --git graylog2-web-interface/src/components/indices/rotation/TimeBasedSizeOptimizingStrategyConfiguration.tsx graylog2-web-interface/src/components/indices/rotation/TimeBasedSizeOptimizingStrategyConfiguration.tsx
index d71cb44359..d8deb101c2 100644
--- graylog2-web-interface/src/components/indices/rotation/TimeBasedSizeOptimizingStrategyConfiguration.tsx
+++ graylog2-web-interface/src/components/indices/rotation/TimeBasedSizeOptimizingStrategyConfiguration.tsx
@@ -24,8 +24,8 @@ import { Input } from 'components/bootstrap';
 import useMaxIndexRotationLimit from 'hooks/useMaxIndexRotationLimit';
 
 export type TimeBasedSizeOptimizingStrategyConfig = {
-  index_lifetime_hard: string,
-  index_lifetime_soft: string,
+  index_lifetime_max: string,
+  index_lifetime_min: string,
   type: string,
 }
 
@@ -42,9 +42,9 @@ const _validationLimit = (durationInDays, rotationLimit) => {
   return durationInDays <= durationToRoundedDays(rotationLimit);
 };
 
-const TimeBasedSizeOptimizingStrategyConfiguration = ({ config: { index_lifetime_hard, index_lifetime_soft }, updateConfig }: Props) => {
-  const [indexLifetimeHard, setIndexLifetimeHard] = useState(index_lifetime_hard);
-  const [indexLifetimeSoft, setIndexLifetimeSoft] = useState(index_lifetime_soft);
+const TimeBasedSizeOptimizingStrategyConfiguration = ({ config: { index_lifetime_max, index_lifetime_min }, updateConfig }: Props) => {
+  const [indexLifetimeHard, setIndexLifetimeHard] = useState(index_lifetime_max);
+  const [indexLifetimeSoft, setIndexLifetimeSoft] = useState(index_lifetime_min);
   const softLifetimeAsDays = durationToRoundedDays(indexLifetimeSoft);
   const hardLifetimeAsDays = durationToRoundedDays(indexLifetimeHard);
   const maxRotationPeriod = useMaxIndexRotationLimit();
@@ -76,7 +76,7 @@ const TimeBasedSizeOptimizingStrategyConfiguration = ({ config: { index_lifetime
     setIndexLifetimeSoft(inputValue);
 
     if (_isValidPeriod(inputValue)) {
-      updateConfig({ index_lifetime_hard: indexLifetimeHard, index_lifetime_soft: inputValue });
+      updateConfig({ index_lifetime_max: indexLifetimeHard, index_lifetime_min: inputValue });
     }
   };
 
@@ -86,7 +86,7 @@ const TimeBasedSizeOptimizingStrategyConfiguration = ({ config: { index_lifetime
     setIndexLifetimeHard(inputValue);
 
     if (_isValidPeriod(inputValue)) {
-      updateConfig({ index_lifetime_soft: indexLifetimeSoft, index_lifetime_hard: inputValue });
+      updateConfig({ index_lifetime_min: indexLifetimeSoft, index_lifetime_max: inputValue });
     }
   };
 
```